lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -2364,6 +2364,59 @@ Value Context::materializeConversion(Type type, Value value, bool isSigned,
     return builder.createOrFold<moore::UIntToRealOp>(loc, type, twoValInt);
   }
 
+  // Handle f64/f32 to time conversion
+  if (isa<moore::TimeType>(type) && isa<moore::RealType>(value.getType())) {
+    auto intType =
+        moore::IntType::get(builder.getContext(), 64, Domain::TwoValued);
+    auto asInt = moore::RealToIntOp::create(builder, loc, intType, value);
```

Comment:
I'm not sure if a 64 bit integer can fully store the value in f64/f32 (i.e. - should the truncation happen during `RealToInt: f64 -> i64` like we're doing now, or during `IntToTime: i1000 -> i64` . Not 100% sure what the semantics of `RealType` -> `IntType` -> `TimeType` are here